tools/nodejs/test/non_inlined_string.js

```diff
@@ -0,0 +1,20 @@
+var duckdb = require('..');
```

Comment:
Can you write this test in typescript please? And follow the file naming pattern

Reply:
All of the tests in here are written in javascript?
I've removed the test file and added it as part of `udf.test.js` instead